plugins/pitchfork.py

```diff
@@ -0,0 +1,36 @@
+import re
+import string
+import textwrap
+
+from errbot import BotPlugin, botcmd
+
+
+class Pitchfork(BotPlugin):
+    """
+    To pitchfork users down to ...
+    """
+
+    @botcmd
+    def pitchfork(self, msg, arg):
+        """
+        To pitchfork user down to ...
+        """
+        match = re.match(r'@?([\w-]+)(?:\s+(?:down\s+)?to\s+(.+))?$',
+                         arg)
+        if match:
+            user = match.group(1)
+            place = match.group(2) if match.group(2) else 'offtopic'
+            return textwrap.dedent((
+                string.Template("""
+                    @$user, you are being pitchforked down to $place
```

Comment:
if its oftopic, we could make it a markdown link? that links to the offtopic channel